pylgbst/messages.py

```diff
@@ -173,7 +173,7 @@ def bytes(self):
         return super(MsgHubAction, self).bytes()
 
     def is_reply(self, msg):
-        assert isinstance(msg, MsgHubAction)
+        assert isinstance(msg, MsgHubAction), type(msg)
```

Comment:
It's good to add some message to it, like `'Unexpected type message type: %s'`. Let's do it good if we got to improving it :)